packages/editor-ui/src/Interface.ts

```diff
@@ -19,6 +18,10 @@ import {
 	IWorkflowSettings as IWorkflowSettingsWorkflow,
 	WorkflowExecuteMode,
 	PublicInstalledPackage,
+	IResourceLocatorResult,
+	INodeTypeNameVersion,
+	ILoadOptions,
+	INodeCredentials,
```

Comment:
unused imports?

Reply:
These are actually in use. I don't know how other branches are building without them though.

Reply:
huh ya, except IResourceLocatorResult is not used